src/main/java/org/mule/runtime/dsl/api/component/config/DefaultComponentLocation.java

```diff
@@ -55,7 +55,7 @@
 @NoExtend
 public class DefaultComponentLocation implements ComponentLocation, Serializable {
 
-  private static final long serialVersionUID = 4958158607813720623L;
+  private static final long serialVersionUID = 4958158607813120623L;
```

Comment:
Is this a "breaking change" that requires to modify the serialVersionUID?
https://docs.oracle.com/javase/6/docs/platform/serialization/spec/version.html#6678